vrk-kpa/suomidigi
# SUOPA-869 legislation card sort

To test: 
- `make fresh`
- Open "Digipalvelulaki" page: https://suomidigi.docker.sh/ohjeet-ja-tuki/lainsaadanto/digipalvelulaki
  - Go to these Legislation taxonomy term pages and edit the order of the nodes: 
     - https://suomidigi.docker.sh/taxonomy/term/41/order
     - https://suomidigi.docker.sh/taxonomy/term/47/order
     - https://suomidigi.docker.sh/taxonomy/term/48/order
  - After you've finished reordering the nodes, refresh the "Digipalvelulaki" page and check that the order of legislation cards have changed
